### ISSUE TITLE
Exception when running `tel status` after rebooting and starting teld.

### ISSUE THREAD

**Issue by pchote** [rockit-astro/mountd-talon#22]
```
Traceback (most recent call last):
  File "/usr/bin/tel", line 519, in <module>
    run_command(status)
  File "/usr/bin/tel", line 98, in run_command
    ret = command()
  File "/usr/bin/tel", line 359, in status
    data = teld.report_status()
  File "/usr/lib/python3.4/site-packages/Pyro4/core.py", line 171, in __call__
    return self.__send(self.__name, args, kwargs)
  File "/usr/lib/python3.4/site-packages/Pyro4/core.py", line 426, in _pyroInvoke
    raise data
AttributeError: 'NoneType' object has no attribute 'date'
 +--- This exception occured remotely (Pyro) - Remote traceback:
 | Traceback (most recent call last):
 |   File "/usr/lib/python3.4/site-packages/Pyro4/core.py", line 1162, in handleRequest
 |     data = method(*vargs, **kwargs)  # this is the actual method call to the Pyro object
 |   File "/usr/bin/teld", line 432, in report_status
 |     alt, az = self.__alt_az_from_ra_dec(self._current_ra, self._current_dec)
 |   File "/usr/bin/teld", line 353, in __alt_az_from_ra_dec
 |     self._observatory.date = ephem.now()
 | AttributeError: 'NoneType' object has no attribute 'date'
 +--- End of remote traceback
```

**Comment by pchote** [comment]
The ephem object is initialized by the first shared memory block parse.  We need to early-return UNKNOWN for the telescope position after a clean boot.

**Comment by pchote** [comment]
Fixed in b85263d2449f4b0b38cb70ecd3d519f9d69b4ff4.
